src/combine.cpp

```diff
@@ -4330,6 +4341,7 @@ static void mark_function(chunk_t *pc)
       //LOG_FMT(LFCN, " -- stopped on %s [%s]\n",
       //        prev->text(), get_token_name(prev->type));
 
+
```

Comment:
Is this allowed by the formatting scripts?

Reply:
I ran the script Run_uncrustify_for_sources and after that i committed.

Reply:
And is it enfoced by the script or simply allowed?

Reply:
i taken the changes suggested by the script.

Reply:
seems to be wrong then but it's unrelated.